tests/functional/olp-cpp-sdk-authentication/AuthenticationCommonTestFixture.cpp

```diff
@@ -28,6 +28,8 @@
 #include "AuthenticationTestUtils.h"
 #include "TestConstants.h"
 
+constexpr auto kDeleteUserWait = std::chrono::seconds(5);
```

Comment:
5 seconds is too much, isn't it?

Reply:
we cannot be sure how much time must pass before sign in later. As discussed in email this is not UI user case to do it immediatly. So in my opinion 5seconds is not too fast for UI case and is not too long to block CI job too much.